src/test/java/teammates/test/cases/browsertests/FeedbackRankQuestionUiTest.java

```diff
@@ -209,6 +209,17 @@ public void testInstructorSubmitAndResultsPage() throws Exception {
     }
 
     @Test
+    public void testInstructorResultsPageForRankRecipientQuestion() throws Exception {
+        ______TS("Rank recipient self excluding instructor results : question");
+
+        InstructorFeedbackResultsPage instructorResultsPage =
+                loginToInstructorFeedbackResultsPageWithViewType("instructor1", "student", false, null);
+        clickAjaxLoadedPanelAndWaitForExpansion(instructorResultsPage, "panelHeading-3", "ajax_auto");
+        clickAjaxLoadedPanelAndWaitForExpansion(instructorResultsPage, "panelHeading-9", "ajax_auto");
+        instructorResultsPage.verifyHtmlMainContent("/instructorFeedbackResultsPageRankRecipientExcludingSelfView.html");
+    }
```

Comment:
Scenario to test this: view by -> question and question type ->rank_recipient. One test case was there and for that `Show Statistics` checkbox was unchecked. So responses were not checked in that.
Moreover there was only one response for one individual and to test average excluding self, at least 2 responses should be present. That's why I made new test.